### DISCUSSION TITLE
Adetailer create different face when using XYZ plot?

### DISCUSSION THREAD

**Discussion by le-khang** [Bing-su/adetailer#211] [General]
Hi Bing-su, I'm using seed 1 and notice that Adetailer doesn't work well with my LORA when I'm using XYZ plot.

Here is the original image

![02723-1](https://github.com/Bing-su/adetailer/assets/35536989/fd842688-2557-4373-ba06-167a0f33c2cc)

Here is the image when I use my LORA and Adetailer (It adds the correct face)

![02726-1](https://github.com/Bing-su/adetailer/assets/35536989/077e8793-d87c-4141-b69a-f75c51b56887)

Here is the image when I use my LORA, Adetailer and XYZ Plot. It doesn't produce the same face as the 2nd picture

![xyz_grid-0091-1](https://github.com/Bing-su/adetailer/assets/35536989/66171f0b-e8e5-4815-a9a1-bd6155ac5c3b)

Adetailer prompt: skinny woman, red hair, solo, looking at viewer, pastel crop top, blunt bangs, mountain landscape, golden hour, depth of field, long hair, upper body <lora:test-000066:1>
In the XYZ plot I write :1,:1.8 or 1>,1.8>
Both aren't working.

![2023-07-15 21_19_52-Stable Diffusion](https://github.com/Bing-su/adetailer/assets/35536989/3843a7b5-252a-4328-92ec-588245bf8656)

**Comment by Bing-su** [comment]
Currently, the ad prompt in the XYZ plot is intended to overwrite it, not replace parts of it.

**Reply by le-khang** [reply]
Ah so that's why. I should try it again. Thank you very much.
